### PR TITLE
Issue 12: Migrating pravega operator charts source code

### LINKED ISSUE #12: Migrate pravega operator and pravega charts
### Description
The charts from the pravega operator repository i.e. the pravega operator charts as well as the pravega charts should be migrated into this repository.

### PR DESCRIPTION
**Pull request by SrishT**
Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

### Change log description
Moves the source code of the charts that were placed within the pravega operator repository into this repository.

### Purpose of the change
Fixes #12 

### How to verify it
Following the steps mentioned in the README should install the pravega operator and pravega successfully.

### CODE REVIEW

**Review by anishakj — CHANGES_REQUESTED** [review]
Changes are looking good,  have seen links are broken at some palces. Could you please fix those?

File: charts/pravega-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Prerequisites
+  - Kubernetes 1.15+ with Beta APIs
+  - Helm 3.2.1+
+  - Cert-Manager v0.15.0+ or some other certificate management solution in order to manage the webhook service certificates. This can be easily deployed by referring to [this](https://cert-manager.io/docs/installation/kubernetes/)
+  - An Issuer and a Certificate (either self-signed or CA signed) in the same namespace that the Pravega Operator will be installed (refer to [this certificate.yaml file](../../deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace)
+
+## Installing Pravega-Operator
+
+> Note: If you are running on Google Kubernetes Engine (GKE), please [check this first](../../doc/development.md#installation-on-google-kubernetes-engine).
+
```
Comment (anishakj) [inline charts/pravega-operator/README.md:14]: correct the link

File: charts/pravega-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+This command deploys a pravega-operator on the Kubernetes cluster in its default configuration. The [configuration](#operator-configuration) section lists the parameters that can be configured during installation.
+
+>Note: If we provide [RELEASE_NAME] same as chart name, deployment name will be same as release-name. But if we are providing a different name for release(other than pravega-operator in this case), deployment name will be [RELEASE_NAME]-[chart-name]. However, deployment name can be overridden by providing --set  fullnameOverride=[DEPLOYMENT_NAME]` along with helm install command
+
+>Note: If the pravega-operator version is 0.4.5, webhookCert.certName and webhookCert.secretName should not be set. Also in this case, bookkeeper operator, cert-manager and the certificate/issuer do not need to be deployed as prerequisites.
+
+## Upgrading Pravega-Operator
+
+For upgrading pravega-operator, please refer [upgrade guide](../../doc/operator-upgrade.md)
```
Comment (anishakj) [inline charts/pravega-operator/README.md:37]: correct the link

File: charts/pravega-operator/README.md (hunk 1 above)
Comment (SrishT) [reply charts/pravega-operator/README.md:14]: done

File: charts/pravega-operator/README.md (hunk 2 above)
Comment (SrishT) [reply charts/pravega-operator/README.md:37]: done

File: charts/pravega-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Deploying Pravega-Operator
+
+Here, we briefly describe how to install [pravega-operator](https://github.com/pravega/pravega-operator) that is used to create/configure/manage Pravega clusters atop Kubernetes.
+
+## Prerequisites
+  - Kubernetes 1.15+ with Beta APIs
+  - Helm 3.2.1+
+  - Cert-Manager v0.15.0+ or some other certificate management solution in order to manage the webhook service certificates. This can be easily deployed by referring to [this](https://cert-manager.io/docs/installation/kubernetes/)
+  - An Issuer and a Certificate (either self-signed or CA signed) in the same namespace that the Pravega Operator will be installed (refer to [this certificate.yaml file](../../deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace)
```
[the remark this answers is not in the document]
Comment (SrishT) [reply charts/pravega-operator/README.md]: done

**Review by anishakj — APPROVED** [review]
LGTM

**Pull request merged**